source/content/guides/composer-convert.md

```diff
@@ -25,34 +36,70 @@ export site=yoursitename
 
 </Alert>
 
-## Checkout a New Branch
+## Add the Pantheon Integrated Composer Upstream in a New Local Branch
 
-You're about to make some massive changes to the codebase. We recommend you to do this work on a new branch, as it might take you some time to complete and rolling back can be complicated:
+You're about to make massive changes to the codebase. We recommend you to do this work on a new branch, as it might take you some time to complete and rolling back changes can be complicated:
```

Comment:
This comment is still true; however, the new instructions are different from the old in now, not only are we making massive changes to the codebase, we are also going to switch our main branch to a completely different git history with no commits in common with the existing repository.